example/use_cases/employer_university_flow/main.go

```diff
@@ -0,0 +1,177 @@
+	employer, err := emp.NewEntity("Employer", "peer")
+	util.HandleExampleError(err, "failed to make employer identity")
+	verifier_did, err := employer.GetWallet().GetDID("main")
+	util.HandleExampleError(err, "failed to create employer")
+
+	example.CustomWriter.Write("Initializing University")
+	university, err := emp.NewEntity("University", "peer")
+	util.HandleExampleError(err, "failed to create university")
+	vcDID, err := university.GetWallet().GetDID("main")
+
+	util.HandleExampleError(err, "falied to initialize verifier")
+	example.CustomWriter.WriteNote(fmt.Sprintf("Initialized Verifier DID: %s and registered it", vcDID))
+	emp.TrustedEntities.Issuers[vcDID] = true
+
+	// Creates the VC
+	example.CustomWriter.Write("Example University Creates VC for Holder")
+	example.CustomWriter.WriteNote("DID is shared from holder")
+	holderDID, err := student.GetWallet().GetDID("main")
+	util.HandleExampleError(err, "failed to store did from university")
+
+	vc, err := emp.BuildExampleUniversityVC(vcDID, holderDID)
+	util.HandleExampleError(err, "failed to build vc")
+
+	// Send to user
+	example.CustomWriter.Write("Example University Sends VC to Holder")
+	student.GetWallet().AddCredentials(*vc)
+	msg := fmt.Sprintf("VC puts into wallet. Wallet size is now: %d", student.GetWallet().Size())
+	example.CustomWriter.WriteNote(msg)
+
+	example.CustomWriter.WriteNote(fmt.Sprintf("initialized verifier DID: %v", verifier_did))
+
+	// 	Presentation Request
+	example.CustomWriter.Write("Employer wants to verify student graduated from Example University. Sends a presentation request")
+	presentationData, err := emp.MakePresentationData("test-id", "id-1")
+	util.HandleExampleError(err, "failed to create pd")
+	if dat, err := json.Marshal(presentationData); err == nil {
+		logrus.Debugf("Presentation Data:\n%v", string(dat))
+	}
+
+	jwk, err := cryptosuite.GenerateJSONWebKey2020(cryptosuite.OKP, cryptosuite.Ed25519)
+	if err != nil {
+		return
+	}
+
+	presentationRequest, _, err := emp.MakePresentationRequest(*jwk, presentationData, holderDID)
+	util.HandleExampleError(err, "failed to make presentation request")
+
+	verifier, err := cryptosuite.NewJSONWebKeyVerifier(jwk.ID, jwk.PublicKeyJWK)
+	util.HandleExampleError(err, "failed to build json web key verifier")
+
+	signer, err := cryptosuite.NewJSONWebKeySigner(jwk.ID, jwk.PrivateKeyJWK, cryptosuite.AssertionMethod)
+	util.HandleExampleError(err, "failed to build json web key signer")
+
+	// 	send the PR back
+	example.CustomWriter.WriteNote("Student returns claims via a Presentation Submission")
+	submission, err := emp.BuildPresentationSubmission(presentationRequest, signer, *verifier, *vc)
+	util.HandleExampleError(err, "failed to buidl presentation submission")
+
+	vp, err := signing.VerifyVerifiablePresentationJWT(*verifier, string(submission))
+	util.HandleExampleError(err, "failed to verify jwt")
+
+	if dat, err := json.Marshal(vp); err == nil {
+		logrus.Debugf("Submission:\n%v", string(dat))
```

Comment:
curious why not `util.HandleExampleError` here?

Reply:
see above comments.